### DISCUSSION TITLE
See the expenses in the categories

### DISCUSSION THREAD

**Discussion by Jim8Knopf** [enrique-lozano/Monekin#203] [Ideas / New features]
Enhancement I'd love  
---

Currently you can't see you're spend expenses in the categories.

Possible solutions
---

It would be a nice feature I saw in an other app. Defiantly not necessary but nice to have. I saw it in a project that also uses Flutter. Maybe this is helpful. [Flow](https://github.com/flow-mn/flow)

![Screenshot_20240924-114209~2](https://github.com/user-attachments/assets/c61f79b2-5023-487c-b38a-114f62f0a52a)

**Comment by enrique-lozano** [comment]
Hello, thank you very much for leaving the idea here.

I think I have an alternative that could be useful for you (under my point of view it is much more practical). In the second tab of the statistics page, you have a graph of expenses and income divided by categories, and immediately after you have a list sorted by the categories where you have spent more. Here is a screenshot (sorry this is in Spanish, but it is the same in any language):

<img src="https://github.com/user-attachments/assets/e6057765-3551-428b-a9e0-e7e7219474d9" width="200">

You can access this view by clicking the arrows on each of the dashboard graphs or from “More > Stats”.

**Reply by Jim8Knopf** [reply]
Thanks a lot for this recommendation. I already was on this page and I think it's really helpful. I just encountered the other view in an other app and thought it might also be interesting.
